src/Renderer/LayeredMaterial.js

```diff
@@ -270,7 +273,9 @@ LayeredMaterial.prototype.setParam = function(param) {
     this.uniforms.nColorLayer.value = param.length;
     for (var l = 0; l < param.length; l++) {
         this.layerIdToIndex[param[l].idLayer] = l;
-        this.paramLayers[l] = new THREE.Vector4(param[l].layerTexturesOffset, param[l].tileMT === 'PM' ? 1 : 0, param[l].visible, param[l].opacity);
+        this.paramLayers[l] = new THREE.Vector4(
+            param[l].layerTexturesOffset,
+            param[l].tileMT === 'PM' ? 1 : 0, param[l].visible, param[l].opacity);
```

Comment:
Could you remove the reference to 'PM' while you're at it? Seems way too specific to be in the material code.

According to the globe fragment shader, parameter.y indicates whether the layer is in wgs84 projection or not. I suggest replacing tileMT by a appropriately-named boolean (or adding a new attribute if you want to keep tileMT).